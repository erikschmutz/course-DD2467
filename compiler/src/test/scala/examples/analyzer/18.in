let p a = a > 2

(*====================
val p : int=>bool
- :int=>bool
====================*)