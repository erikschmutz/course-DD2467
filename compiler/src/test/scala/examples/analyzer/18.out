AnalyzerResult(
    Function(
        Integer(),Bool()
    ),
    Enviroment(
        List(
            ValEntry(
                Identifier(p),
                Function(
                    Integer(),
                    Bool()
                )
            )
        )
    )
)